### PR TITLE
More fixes for image styling

### PR DESCRIPTION
**Pull request by LizBaker**
This should fix the image issues we were seeing in collapsers, as well as the styling and functional issues with tech tiles. 
examples: 
`/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/`
`/docs/distributed-tracing/enable-configure/language-agents-enable-distributed-tracing/#update-agents`

### CODE REVIEW

**Comment by roadlittledawn** [comment]
Collapsers on this page now look good! https://build-4cb118a9-ebcb-4bb5-9f36-9c34d18db0c6.gtsb.io/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/

Couple things I noticed on quick look:
- [Code Stream doc with broken images](https://build-4cb118a9-ebcb-4bb5-9f36-9c34d18db0c6.gtsb.io/docs/codestream/codestream-integrations/slack-integration)
  - At quick glance not sure why these images are broken
- the `TechTile` components on this [page](https://build-4cb118a9-ebcb-4bb5-9f36-9c34d18db0c6.gtsb.io/docs/distributed-tracing/enable-configure/language-agents-enable-distributed-tracing/) still seem a bit funky

File: src/components/MDXContainer.js [hunk 1]
```diff
@@ -19,14 +19,14 @@ const defaultComponents = {
...
+        width={props.width ? props.width : 'inherit'}
         src={props.src}
         alt={props.alt ? props.alt : 'Docs site'}
         title={props.title}
         style={
           props.style
-            ? { ...props.style, margin: '0 0.25rem' }
-            : { margin: '0 0.25rem' }
+            ? { ...props.style, margin: '0 0.25rem', maxWidth: '100%' }
+            : { margin: '0 0.25rem', maxWidth: '100%' }
```
Comment (caylahamann) [inline src/components/MDXContainer.js:29]: what does this max-width do?

**Comment by caylahamann** [comment]
> * [Code Stream doc with broken images](https://build-4cb118a9-ebcb-4bb5-9f36-9c34d18db0c6.gtsb.io/docs/codestream/codestream-integrations/slack-integration)
>   
>   * At quick glance not sure why these images are broken

It appears that those weren't turned into `jsx` tags, but remain regular ol' markdown images

**Comment by LizBaker** [comment]
ah interesting, the TechTiles look different in a prod build vs locally, I'll poke at that

File: src/components/MDXContainer.js (hunk 1 above)
Comment (LizBaker) [reply src/components/MDXContainer.js:29]: why.. nothing.. 😂 i'll remove that

**Review by roadlittledawn — APPROVED** [review]
Looks good! I checked bunch of instances of:
- inline images
- block images
- images used in `Collapser` title props
- images used in `TechTile` icon props

**Pull request merged**